input/input.go

```diff
@@ -0,0 +1,252 @@
+/*
+Package Input reads config file and populates the data in Config struct.
```

Comment:
file name should be config.go as it's handling config parsing and validation.

Reply:
its input.go , as in future we may add other input method to it